[2025-01-07T00:02:25.716+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_insert_log.ft_balance_f manual__2025-01-07T00:01:59.393323+00:00 [queued]>
[2025-01-07T00:02:25.876+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_insert_log.ft_balance_f manual__2025-01-07T00:01:59.393323+00:00 [queued]>
[2025-01-07T00:02:25.886+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-01-07T00:02:26.102+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_balance_f> on 2025-01-07 00:01:59.393323+00:00
[2025-01-07T00:02:26.123+0000] {standard_task_runner.py:60} INFO - Started process 1072 to run task
[2025-01-07T00:02:26.147+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_insert_log', 'ft_balance_f', 'manual__2025-01-07T00:01:59.393323+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/dag_insert_log.py', '--cfg-path', '/tmp/tmpwxkfvqco']
[2025-01-07T00:02:26.163+0000] {standard_task_runner.py:88} INFO - Job 80: Subtask ft_balance_f
[2025-01-07T00:02:26.354+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_insert_log.ft_balance_f manual__2025-01-07T00:01:59.393323+00:00 [running]> on host 22295d32c47d
[2025-01-07T00:02:26.941+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='saida' AIRFLOW_CTX_DAG_ID='dag_insert_log' AIRFLOW_CTX_TASK_ID='ft_balance_f' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T00:01:59.393323+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T00:01:59.393323+00:00'
[2025-01-07T00:02:27.083+0000] {base.py:83} INFO - Using connection ID '***-db' for task execution.
[2025-01-07T00:02:27.697+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-07T00:02:27.835+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_insert_log, task_id=ft_balance_f, execution_date=20250107T000159, start_date=20250107T000225, end_date=20250107T000227
[2025-01-07T00:02:28.023+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-07T00:02:28.345+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
